frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepTable.tsx

```diff
@@ -26,13 +26,18 @@ function getStepColor(step: FlattenedFunnelStep, isBreakdown?: boolean): string
     return getColor(step, 'var(--text-default)', isBreakdown)
 }
 
+function isBreakdownChildType(
```

Comment:
I'm confused by the name here

Reply:
I could use some typescript magic here but this utility function just type guards the breakdown parameter because in `formatBreakdownLabel` the linter was complaining. When we're looking at breakdown child rows, the breakdown parameter should be string/number/undefined and not an array